### ISSUE TITLE
Exception on multi-touch for `useDrag` and v10

### ISSUE THREAD

**Issue by BowlingX** [pmndrs/use-gesture#323]
**Describe the bug**
When using multi-touch and dragging an element, I receive an exception `_event$currentTarget.contains is not a function`.

```
TypeError: _event$currentTarget.contains is not a function. (In '_event$currentTarget.contains(e.target)', '_event$currentTarget.contains' is undefined)
```

![image](https://user-images.githubusercontent.com/215671/123519519-02863180-d6ac-11eb-9669-8857d3f820da.png)

**Sandbox or Video**
https://9bk8y.csb.app/
https://codesandbox.io/s/react-spring-carousel-9bk8y?file=/src/Slider.tsx

Can be reproduced with iOS (tested on 14) Safari and using one, then 2 fingers to drag the carousel.

**Information:**
 - React Use Gesture version: v.10.0.0-beta.14
 - Device: Safari IOS
 - OS: iOS
 - Browser Safari

**Checklist:**
  - [x ] I've read the [documentation](https://use-gesture.netlify.app/).

**Comment by dbismut** [comment]
Should be fixed in 10.0.0-beta.15, let me know!

**Comment by BowlingX** [comment]
Fixed :) Thank you!!
